app/views/registerusers/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>New User</h1>
+<%if flash[:errors]%>
+  <%flash[:errors].each do |error|%>
+    <p class="text-center" style='color: red'><%= error%></p>
+  <%end%>
+<%end%>
+<div class=" p-5" >
+  <%= form_with model: @registeruser do |f| %>
+    <%= f.label :first_name, "First Name :", class:"mt-3" %>
+    <%= f.text_field :first_name ,placeholder: "Enter Your First Name",class:"mb-2 form-control" %>
+    <br>
+    <%= f.label :last_name, "Last Name :" %>
+    <%= f.text_field :last_name ,placeholder: "Enter Your Last Name",class:"mb-2 form-control" %>
+    <br>
+    <%= f.label :email, "E-mail ID :" %>
+    <%= f.email_field :email ,placeholder: "Enter Your E-mail",class:"mb-2 form-control" %>
+    <br>
+    <%= f.label :password, "Password :" %>
+    <%= f.password_field :password ,placeholder: "Enter Your Password",class:"mb-2 form-control" %>
+    <br>
+    <%= f.label :subscription, "Do you want a Subscription?" %> &nbsp;
+    <%= f.check_box :subscription,class:"form-check-input" %>
+    <br>
+    <%= f.label :subscription_email, "Subscription Email :", class:"mt-3" %>
+    <%= f.email_field :subscription_email ,placeholder: "Enter E-mail for subscription",class:"mb-2 form-control" %>
+    <br>
+    <div class="d-flex">
+    <%= f.submit "Save User", class:"btn-success  mt-4 me-2  border-0 p-2 rounded-pill center" %>
+    <% end %>
+    <%= button_to "View all Users", registerusers_path,method: :get,class:"btn-primary center mt-4 border-0 p-2 rounded-pill" %>
+    </div>
+</div>
```

Comment:
End of line missing 
add end of line on every file where it is missing.

Reply:
Changes done, Can you please check.